src/components/map/MapComponent.vue

```diff
@@ -192,6 +193,20 @@ async function displaySolarPanelLayer() {
   await zoomToSolarPanel(rennesApp)
 }
 
+function IsPanelVisibleOnStep() {
```

Comment:
You can put this function in one service (You don't have to create a new one), because this function is use in two different file